packages/yourloops/components/patient/table.tsx

```diff
@@ -47,51 +48,49 @@ import { PatientTableSortFields } from './models/enums/patient-table-sort-fields
 import { Patient } from '../../lib/patient/models/patient.model'
 import { SortDirection } from './models/enums/sort-direction.enum'
 
-const patientListStyle = makeStyles(
-  (theme: Theme) => {
-    return {
-      flagSort: {
-        width: '100%',
-        justifyContent: 'center'
-      },
-      pagination: {
-        '& .MuiTablePagination-spacer': {
-          display: 'none'
-        },
-        '& .MuiTablePagination-caption:last-of-type': {
-          marginLeft: 'auto'
-        }
-      },
-      infoIcon: {
-        marginLeft: theme.spacing(1),
-        marginRight: '2px'
-      },
-      tableContainer: {
-        boxShadow: theme.shadows[2]
-      },
-      tableRowHeader: {
-        padding: 0,
-        height: '62px'
-      },
-      tableCellHeader: {
-        backgroundColor: theme.palette.common.white,
-        fontSize: '15px',
-        fontWeight: 600,
-        padding: 0,
-        paddingLeft: '11px'
-      },
-      alertTimeTargetHeader: {
-        maxWidth: '210px'
+const patientListStyle = makeStyles({ name: 'ylp-hcp-patients-table' })((theme: Theme) => {
+  return {
+    flagSort: {
+      width: '100%',
+      justifyContent: 'center'
+    },
+    pagination: {
+      '& .MuiTablePagination-spacer': {
+        display: 'none'
       },
-      tableHeaderIcon: {
-        width: '56px',
-        padding: 0
+      '& .MuiTablePagination-caption:last-of-type': {
+        marginLeft: 'auto'
       }
+    },
+    infoIcon: {
+      marginLeft: theme.spacing(1),
+      marginRight: '2px'
+    },
+    tableContainer: {
+      boxShadow: theme.shadows[2]
+    },
+    tableRowHeader: {
+      padding: 0,
+      height: '62px'
+    },
+    tableCellHeader: {
+      backgroundColor: theme.palette.common.white,
+      fontSize: '15px',
+      fontWeight: 600,
+      padding: 0,
+      paddingLeft: '11px'
+    },
+    alertTimeTargetHeader: {
+      maxWidth: '210px'
+    },
+    tableHeaderIcon: {
+      width: '56px',
+      padding: 0
     }
-  },
-  { name: 'ylp-hcp-patients-table' }
-)
-export const patientListCommonStyle = makeStyles(() => {
+  }
+})
+// TODO jss-to-tss-react codemod: usages of this hook outside of this file will not be converted.
```

Comment:
? Is there are JIRA ticket? I don't understand the comment either

Reply:
Just forgot to remove it, it was generated by the tss-react utility to migrate from jss to tss